client/src/components/loginComponents/LoginForm.jsx

```diff
@@ -19,8 +20,32 @@ const LoginForm = () => {
     setModalOpen(false);
   };
   //----------------Embedded Functions -------------------
-
-
+  const handlePlainLogin = () => {
+    setLoading(true);
+    login(email, password)
+    .then(() => {
+      setLoading(false);
+      if (accountType = "seeker") {
+        navigate("./Seeker.js", {replace: true});
+      } else {
+        navigate("./Recruiter.js", {replace: true});
+      }
+    })
+    .catch(err => {console.log('There was an error logging in: ', err)})
+  }
+  const handleGoogleLogin = () => {
+    setLoading(true);
+    googleLogin()
+    .then(() => {
+      setLoading(false);
+      if (accountType = "seeker") {
+        navigate("/seeker", {replace: true});
+      } else {
+        navigate("/recruiter", {replace: true});
+      }
+    })
+    .catch(err => {console.log('There was an error logging in: ', err)})
```

Comment:
I'm getting an error here when attempting to sign in with Google on local

Error:
`
There was an error logging in:  ReferenceError: accountType is not defined at eval (LoginForm.jsx:84:23)
`

Attempting to login with Google appears to correctly make an account correctly, but does not assign a `seeker` or `recruiter` value to it

This also appears to prevent that gmail address from being used to create a new account since it is already in use